onnx_xla/translations/translate_add.cc

```diff
@@ -0,0 +1,18 @@
+#include "onnx_xla/operator_registry.h"
```

Comment:
all the elementwise binary operators are similar, maybe use a macro?

Reply:
Yeah makes sense with Sub and Div - changed